django/core/serializers/python.py

```diff
@@ -54,6 +54,13 @@ def handle_field(self, obj, field):
             self._current[field.name] = field.value_to_string(obj)
 
     def handle_fk_field(self, obj, field):
+        # If not using natural primary keys, then there's no need to output
+        # the _ptr field in multi-table inheritance models, since the
+        # PK of the subclass instance is garanteed to be the same as the PK
+        # of the superclass instance.
+        if not self.use_natural_primary_keys and field == obj._meta.pk:
```

Comment:
Perhaps this should be a method on the `base.Serializer` class so it can be reused?